src/hotspot/share/runtime/mutexLocker.cpp

```diff
@@ -372,3 +372,22 @@ void print_owned_locks_on_error(outputStream* st) {
   }
   if (none) st->print_cr("None");
 }
+
+void unlock_locks_on_error(JavaThread* thread) {
```

Comment:
Nit: s/thread/current/ as we must be applying this to the current thread.

Reply:
make sense. I will make them consistent.
Should be `thread` because it's not necessarily be current().